clusterloader2/cmd/clusterloader.go

```diff
@@ -95,6 +96,8 @@ func validateClusterFlags() *errors.ErrorList {
 }
 
 func initFlags() {
+	const defaultLoggingLevel = "2"
```

Comment:
This is default anyway - no need to do that. Please remove

Reply:
OK, but note that the default is 0.